plugin/goto.py

```diff
@@ -58,7 +58,7 @@ def handle_response(self, response: Any, side_by_side: bool) -> None:
         if window is None:
             return
         if response:
-            if any(view.sel()):
+            if len(view.sel()) > 0:
```

Comment:
`if view.sel()` would be good enough. :)

Reply:
No, sublime.Selection has a custom `__bool__` method. It returns true if and only if the view is valid.

Reply:
Hmm... `bool(view.sel())` == True

Reply:
The bool of Selection object is not relevant when using just `if view.sel()`. It would just check that the list is non-empty, ignoring the bool operator of the Selection object.

Reply:
Really? I've learned something new. I thought Python would evaluate that as

```python
if view.sel().__bool__()
```
(for any object that has a custom `__bool__`)

Reply:
Ah, sorry, my bad. I thought that `view.sel()` is a list of `Selection` objects while it's actually the `Selection` object. So you are right. :)